specs/validator/README.md

```diff
@@ -224,9 +228,9 @@ Parameters
 ``` json
 {
   "name": "healthbot_5_1",
-  "external_url": "https://testnet.tableland.network/tables/healthbot_5_1",
-  "animation_url": "https://render.tableland.xyz/anim/?chain=1&id=1",
-  "image": "https://render.tableland.xyz/healthbot_5_1",
+  "external_url": "https://testnets.tableland.network/api/v1/tables/healthbot_5_1",
```

Comment:
@brunocalza general question—i'm assuming _all_ docs should now use `https://tables.tableland.xyz` over `https://render.tableland.xyz`? i.e., drop mentions of `https://render.tableland.xyz` entirely. 

i'll update this info in various docs snippets but just wanna double check before i make any changes.

Reply:
oh—and i guess we should change the example table in all of these, right? it's the deprecated goerli's `healthbot_5_1`.

Reply:
> @brunocalza general question—i'm assuming _all_ docs should now use `https://tables.tableland.xyz` over `https://render.tableland.xyz`? i.e., drop mentions of `https://render.tableland.xyz` entirely.
> 
> i'll update this info in various docs snippets but just wanna double check before i make any changes.

yeah, we should change in all places. I can search for all places and do another commit in this PR, wdyt? I'll update the to sepolia as well

Reply:
sure, that works for me!